dev/compare_models.py

```diff
@@ -305,7 +302,7 @@ def compute_statistics(results_df, n_iterations):
     results_df = results_df.reset_index()
     results_df = results_df.sort_values(by=['best_validation_loss'])
 
-    results_df.to_csv("output_df.csv")
+    results_df.to_csv("detailled_results.csv")
```

Comment:
Detailed?

Reply:
I named `detailled_results` because it has the results for all iterations, I thought `output_df` was too generic since avg, std and p-values are also outputs, open to suggestions :)

Reply:
i think Charley's suggestion was about the typo: detailled --> detailed

Reply:
Oups yep thanks!